tests/client/core/reducers/test_authentication.js

```diff
@@ -43,7 +43,7 @@ describe('authentication reducer', () => {
       });
       assert.throws(
         () => setAndReduceToken(token),
-        Error, /Error parsing auth token .* JSON\.parse: unexpected character/);
+        Error, /Error parsing auth token "{"malformed JSON"}/);
```

Comment:
These kind of changes were necessary since the exception messages are subtly different under node.